Build failed:
```
/__w/1/s/src/tests/tracing/runtimeeventsource/NativeRuntimeEventSourceTest.cs(27,34): error CS0103: The name 'ArrayPool' does not exist in the current context [/__w/1/s/src/tests/tracing/runtimeeventsource/nativeruntimeeventsource.csproj] [/__w/1/s/src/tests/build.proj]
##[error]src/tests/tracing/runtimeeventsource/NativeRuntimeEventSourceTest.cs(27,34): error CS0103: The name 'ArrayPool' does not exist in the current context [/__w/1/s/src/tests/tracing/runtimeeventsource/nativeruntimeeventsource.csproj]
/__w/1/s/src/tests/tracing/runtimeeventsource/NativeRuntimeEventSourceTest.cs(28,60): error CS1061: 'byte[]' does not contain a definition for 'Lenth' and no accessible extension method 'Lenth' accepting a first argument of type 'byte[]' could be found (are you missing a using directive or an assembly reference?) [/__w/1/s/src/tests/tracing/runtimeeventsource/nativeruntimeeventsource.csproj] [/__w/1/s/src/tests/build.proj]
##[error]src/tests/tracing/runtimeeventsource/NativeRuntimeEventSourceTest.cs(28,60): error CS1061: 'byte[]' does not contain a definition for 'Lenth' and no accessible extension method 'Lenth' accepting a first argument of type 'byte[]' could be found (are you missing a using directive or an assembly reference?) [/__w/1/s/src/tests/tracing/runtimeeventsource/nativeruntimeeventsource.csproj]
```

--- a/src/tests/tracing/runtimeeventsource/NativeRuntimeEventSourceTest.cs
+++ b/src/tests/tracing/runtimeeventsource/NativeRuntimeEventSourceTest.cs
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Buffers;
 using System.Diagnostics;
 using System.Diagnostics.Tracing;
 using System.IO;
@@ -25,7 +26,7 @@ namespace Tracing.Tests
             // Access ArrayPool.Shared.Rent() before the test to avoid the deadlock reported
             // in https://github.com/dotnet/runtime/issues/86233
             byte[] localBuffer = ArrayPool<byte>.Shared.Rent(10);
-            Console.WriteLine($"buffer length={localBuffer.Lenth}");
+            Console.WriteLine($"buffer length={localBuffer.Length}");
 
             // Create deaf listener
             Listener.Level = EventLevel.Critical;

--- a/src/tests/tracing/runtimeeventsource/NativeRuntimeEventSourceTest.cs
+++ b/src/tests/tracing/runtimeeventsource/NativeRuntimeEventSourceTest.cs
@@ -24,7 +24,9 @@ namespace Tracing.Tests
         private static int Main()
         {
             // Access ArrayPool.Shared.Rent() before the test to avoid the deadlock reported
-            // in https://github.com/dotnet/runtime/issues/86233
+            // in https://github.com/dotnet/runtime/issues/86233. This is a real issue,
+            // but only seen if you have a short lived EventListener and create EventSources
+            // in your OnEventWritten callback so we don't expect customers to hit it.
             byte[] localBuffer = ArrayPool<byte>.Shared.Rent(10);
             Console.WriteLine($"buffer length={localBuffer.Length}");
 

